config.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Module for parsing configuration file.
+
+Configuration file should have name 'config.ini'.
+Also 'config.ini' file should be in the same catalog with config.py file.
+
+Example of configuraion file:
+[DATABASE]
+uri: mysql://root:123@172.17.0.3/CYCLINGDB
```

Comment:
I recommend you change config file into next format:
[DataBase]
ENGINE = mysql
DB_USER = admin
PASSWORD = passpass
HOST = localhost
PORT = 8029
NAME = myDataBase

Please change this into next version.